clients/devtools/devtools-entry.js

```diff
@@ -82,6 +82,30 @@ function lookupCanonicalLocale(locales) {
   return lookupLocale(locales, getCanonicalLocales());
 }
 
+/**
+ * TODO: Expose api directly when DevTools usage is updated.
+ * @param {{page: LH.Puppeteer.Page, url: string, config?: LH.Config.Json, flags?: LH.Flags}} args
+ */
+function runLighthouseNavigation({page, url, ...options}) {
+  return navigation(page, url, options);
+}
```

Comment:
We can keep these functions forever, or we could expose `navigation` as `runLighthouseNavigation2` to help transition off this API.